### PR TITLE
Component

### PR DESCRIPTION
**Pull request by KBhautik6**
**Checking list** 

- [x] Check if you check coding?
- [x] Checking the code is duplicate or not?
- [ ] Is code is check by shubhum sir or ankit sir?
- [x] Are you going to merge this or not?

### Creating the Primary secondary sucesson and warning button using storybook
Screenshot 2023-12-13 at 3 57 10 PM

### Creating the Primary secondary sucesson and warning button with **hover** using storybook 
Screenshot 2023-12-13 at 5 34 53 PM

### Creating the save button 
Screenshot 2023-12-13 at 5 47 49 PM

### creating the save with **hover ** 
Screenshot 2023-12-13 at 6 11 41 PM

### Creating the text-button of primary, secondary , successon and warning
Screenshot 2023-12-13 at 6 11 29 PM

### CODE REVIEW

File: dist/main.js
Comment (tarakhant1297) [inline dist/main.js:1]: @KBhautik6 put dist in gitignore

File: dist/index.html
Comment (tarakhant1297) [inline dist/index.html:1]: @KBhautik6 put dist in gitignore

File: src/Button/Button.Styled.js [hunk 1]
```diff
@@ -0,0 +1,173 @@
...
+export const StyledButton = styled.button`
+    color: #F6F9FC;
+    font:14px 'Arial';
+    border: none;
+    background-color:#FF4E54;
+    padding:11px 28px;
+    margin:10px;
+    cursor: pointer;
+`;
+export const SecondButton = styled.button`
```
Comment (tarakhant1297) [inline src/Button/Button.Styled.js:14]: @KBhautik6 use prefix `Styled` for this

File: src/Button/Button.Styled.js [hunk 2]
```diff
@@ -0,0 +1,173 @@
+import styled from 'styled-components';
+
+
+export const StyledButton = styled.button`
+    color: #F6F9FC;
+    font:14px 'Arial';
+    border: none;
+    background-color:#FF4E54;
```
Comment (tarakhant1297) [inline src/Button/Button.Styled.js:8]: @KBhautik6 read color from theme

File: src/Button/Button.Styled.js [hunk 3]
```diff
@@ -0,0 +1,173 @@
...
+export const SecondButton = styled.button`
+    color: #4a4a68;
+    margin:10px;
+    border: none;
+    font:14px 'Arial';
+    background-color:#F3F4F6;
+    padding:11px 28px;
+    cursor: pointer;
+`;
+export const ThridButton=styled.button`
```
Comment (tarakhant1297) [inline src/Button/Button.Styled.js:24]: @KBhautik6 add one line above each styled

File: src/Button/Button.Styled.js [hunk 4]
```diff
@@ -0,0 +1,173 @@
...
+export const Primarybuttonhover=styled.button`
+    border:1px solid black;
+    color: #FF4E54;
+    font:14px 'Arial';
+    border: none;
+    background-color:#f6f9fc;
+    padding:11px 28px;
+    margin:10px;
+    cursor: pointer;
+    &:hover,
```
Comment (tarakhant1297) [inline src/Button/Button.Styled.js:56]: @KBhautik6 add one line above this hove

File: src/Button/Button.Styled.js [hunk 5]
```diff
@@ -0,0 +1,173 @@
...
+`
+export const SecondButtonhover = styled.button`
+    color: #4a4a68;
+    margin:10px;
+    border: none;
+    font:14px 'Arial';
+    background-color:#f6f9fc;
+    padding:11px 28px;
+    cursor: pointer;
+    &:hover,
```
Comment (tarakhant1297) [inline src/Button/Button.Styled.js:72]: @KBhautik6 add one line above this

File: src/Button/Button.Styled.js [hunk 6]
```diff
@@ -0,0 +1,173 @@
...
+    active{
+        background-color:#4CAF50;
+        color:#f6f9fc;
+    }
+`
+
```
Comment (tarakhant1297) [inline src/Button/Button.Styled.js:110]: @KBhautik6 remove extra space

File: src/Button/Button.Styled.js [hunk 7]
```diff
@@ -0,0 +1,173 @@
...
+
+
+export const Save_hover=styled.button`
```
Comment (tarakhant1297) [inline src/Button/Button.Styled.js]: @KBhautik6 give proper name to this styled

File: src/Button/Button.Styled.js [hunk 8]
```diff
@@ -0,0 +1,173 @@
...
+  font-size: 16px;
+  cursor: pointer;
+  &:hover,
+  &:active {
+    background-color: #ff4e54;
+    color: #ffffff;
+  }
+`
+
+export const Save_new=styled.button`
```
Comment (tarakhant1297) [inline src/Button/Button.Styled.js]: @KBhautik6 same here

File: src/Button/Button.js [hunk 9]
```diff
@@ -0,0 +1,76 @@
...
+        WARNING BUTTON
+      </FourthButtonhover>
+    </div>
+  );
+};
+export const Icon=()=>{
+  return(
+    <div>
+      <Save_hover>
+      <svg
```
Comment (tarakhant1297) [inline src/Button/Button.js:34]: @KBhautik6 create svg file for this image and use here

File: src/stories/Button.stories.js [hunk 10]
```diff
@@ -0,0 +1,15 @@
+import React from "react";
+import {Button,Hover,Icon,Text_Button}  from "../Button/Button";
+
+// import Buttonhover from "../Button/Buttonhover";
```
Comment (tarakhant1297) [inline src/stories/Button.stories.js:4]: @KBhautik6 remove commented line

File: src/stories/assets/accessibility.png
Comment (tarakhant1297) [inline src/stories/assets/accessibility.png:1]: @KBhautik6 remove extra images

File: src/Button/Button.Styled.js [hunk 11]
```diff
@@ -0,0 +1,173 @@
...
+    active{
+        background-color:#4CAF50;
+        color:#f6f9fc;
+    }
+`
+
+
+
+export const Save_hover=styled.button`
+  border: 2px solid; // Add a unit (e.g., pixels)
```
Comment (tarakhant1297) [inline src/Button/Button.Styled.js:112]: @KBhautik6 add color for border from theme

**Review by tarakhant1297 — CHANGES_REQUESTED** [review]

File: src/Button/Button.Styled.js (hunk 1 above)
Comment (KBhautik6) [reply src/Button/Button.Styled.js:14]: Basically it's function given by styled - component so , I use this thing

File: src/Button/Button.Styled.js (hunk 2 above)
Comment (KBhautik6) [reply src/Button/Button.Styled.js:8]: Ok, I will make new theme file for this and import on button.style.js and use that ::root file at here and from next time i will take of it.

File: src/Button/Button.Styled.js (hunk 3 above)
Comment (KBhautik6) [reply src/Button/Button.Styled.js:24]: Ok done

File: src/Button/Button.Styled.js (hunk 4 above)
Comment (KBhautik6) [reply src/Button/Button.Styled.js:56]: Done

File: src/Button/Button.Styled.js (hunk 5 above)
Comment (KBhautik6) [reply src/Button/Button.Styled.js:72]: Done

File: src/Button/Button.Styled.js (hunk 6 above)
Comment (KBhautik6) [reply src/Button/Button.Styled.js:110]: Done

File: src/Button/Button.Styled.js (hunk 11 above)
Comment (KBhautik6) [reply src/Button/Button.Styled.js:112]: okay

File: dist/index.html
Comment (KBhautik6) [reply dist/index.html:1]: Okay

File: src/stories/Button.stories.js (hunk 10 above)
Comment (KBhautik6) [reply src/stories/Button.stories.js:4]: Okay

File: src/stories/assets/accessibility.png
Comment (KBhautik6) [reply src/stories/assets/accessibility.png:1]: Okay i will put in .gitignore

File: src/Button/Button.Styled.js (hunk 7 above)
Comment (KBhautik6) [reply src/Button/Button.Styled.js]: Done

File: src/Button/Button.js (hunk 9 above)
Comment (KBhautik6) [reply src/Button/Button.js:34]: I try but it show some error so i directly use the svg again 

ERROR in ./src/image/logo.svg
Module build failed (from ./node_modules/@svgr/webpack/lib/index.js):
SyntaxError: unknown: Namespace tags are not supported by default. React's JSX doesn't support namespace tags. You can set `throwIfNamespace: false` to bypass this warning.

**Review by KBhautik6 — COMMENTED** [review]
complete till badges

**Pull request merged**